src/components/Search.js

```diff
@@ -28,14 +30,26 @@ const Search = props => (
                 />
               </td>
               <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
+                <SearchButton />
+              </td>
+              <td>
```

Comment:
There's a repeating pattern here, maybe something we want to make into a component or 2?